packages/react/src/components/Carousel/carousel.scss

```diff
@@ -60,6 +60,15 @@
     z-index: 1;
     position: absolute;
     width: 100%;
-    bottom: calc(42px + 1em);
+    // TODO: Review this.
```

Comment:
Whats the reason for the absolute positioning?

Reply:
Absolute positioning was required for the navigation buttons in the Mobile screen. 

<img width="500" alt="Screenshot 2023-03-07 at 11 09 54" src="https://user-images.githubusercontent.com/67315176/223331392-92fd4ba8-8358-45c6-b215-12a82a67a5a1.png">

Reply:
The design looks like a UI issue to me. Shall we check what we can do to improve?

Reply:
A discussion is being done on a better design for the buttons. A fix is added until the decision is finalized - https://github.com/wso2/oxygen-ui/pull/58/commits/ec27f4f8cbb69d032576bc3df96e1ea22b758348